styles.css

```diff
@@ -614,4 +604,48 @@ header {
     display: flex;
     flex-direction: column;
   }
+
+  /* Contact Form */
+  .contact-form-section {
+    position: absolute;
+    padding: 10% 30%;
+    gap: 52px;
+    height: 846px;
+    top: 70%;
+  }
+
+  .form-div {
+    gap: 27px;
+    margin: auto;
+  }
+
+  .text-input,
+  .email-input {
+    height: 48px;
+    background: #fff;
+    border: 1px solid #cfd8dc;
+    border-radius: 2px;
+    padding-left: 16px;
+  }
+
+  .textarea-input {
+    height: 170px;
+    background: #fff;
+    border: 1px solid #cfd8dc;
+    border-radius: 2px;
+    padding: 15px 38px 15px 16px;
+  }
+
+  .contact-form-bg {
+    background-image: url(./images/contact-form-bg@2x.png);
+    background-size: contain;
+    background-repeat: no-repeat;
+    left: 2.5%;
+    height: 95%;
+    width: 95%;
+  }
+
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** Good job all changes mentioned by previous reviewers. However, I noticed that there is **one important thing they forgot to check**. Right now, your form inputs only work on mobile, user can write something inside only if he is in mobile version but not on **Desktop version**.  I think it would be better if you fix it and this because you are using your `background-image: url(./images/contact-form-bg@2x.png);` in the `<div class="contact-form-bg"></div>` whereas that it should be the  `background-image` of the entire section(i.e. in the ` <section class="contact-form-section flex-column"> ... </section>`). I recommend you move the ``background-image: url(./images/contact-form-bg@2x.png);`` in the `.contact-form-section { ... }` class then and style it correctly to match the Figma design. **Also remem**
 
```HTML
// Bad
  .contact-form-bg {
    background-image: url(./images/contact-form-bg@2x.png);
    ...
  }

// Good
 .contact-form-section {
    background-image: url(./images/contact-form-bg@2x.png);
    ...
  }
```

### IMPORTANT:
**I recommend you fix this issue before merging your pull request as it was not mentioned previous reviewers but it should be required**

Reply:
Thank you for your observation, I have made the changes.